compoments/report-repair/summary.js

```diff
@@ -14,20 +16,34 @@ const Summary = ({values, submit, goToStep}) => {
   const repairDetailsSummary = [
     { pageName:'Where is the problem?', value: values.repairLocation?.display, link:'repair-location'},
     { pageName:'What is the problem?', value: values.repairProblem?.display, link: 'repair-problems'},
-    { pageName:'What best describes the problem?', value: values['repairProblemBestDescription']?.display, link: 'repair-problem-best-description'},
     { pageName:'Description', value: values.description?.text, link:'repair-description'}
   ]
-  if (!isMvpReleaseVersion()) {
+  if (!isMvpReleaseVersion() && repairType != 'Communal') {
```

Comment:
So far the result of `getRepairType` has not been used to make comparisons, i.e. `!= 'Communal'`.

There are multiple instances within these changes which check the value against a [Magic String](https://en.wikipedia.org/wiki/Magic_string).

It would be best to introduce a function that evaluates to true/false indicating the result of a comparison, i.e. `isCommunal`.

The comparison is currently made multiple times, whereas it could be made once and reused.
After the refactoring, it would be best to call the function once and store the result for reuse.

Reply:
`isCommunalRepairType` function created.